app/uk/gov/hmrc/emcstfefrontend/models/response/GetMovementResponse.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emcstfefrontend.models.response
+
+import play.api.libs.json.{Json, OFormat}
+
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+
+case class GetMovementResponse(
+                                localReferenceNumber: String,
+                                eadStatus: String,
+                                consignorName: String,
+                                dateOfDispatch: LocalDate,
+                                journeyTime: String,
+                                numberOfItems: Int
+                              ) {
+  def formattedDateOfDispatch: String = {
+    val f = DateTimeFormatter.ofPattern("dd MMMM yyyy")
+    f.format(dateOfDispatch)
+  }
```

Comment:
This could be moved out to a common object, as date formatting sounds like something that will be re-used. Maybe we could have a `DateUtils` trait that contained an implicit class which extended the functions available to dates when mixed into classes.

Something like:
```
trait DateUtils {

  implicit class LocaDateExtensions(date: Date) {
     def formatDateForUIOutput() = {
          val f = DateTimeFormatter.ofPattern("dd MMMM yyyy")
          f.format(dateOfDispatch)
     }
  }
}
```

Then if that trait was mixed in where needed you could just call `dateOfDispatch.formatDateForUIOutput()`